packages/terra-signature/src/Signature.jsx

```diff
@@ -3,18 +3,23 @@ import PropTypes from 'prop-types';
 import 'terra-base/lib/baseStyles';
 import styles from './Signature.scss';
 
-const LineWidths = {
+const LINEWIDTHS = {
   EXTRA_FINE: 1,
   FINE: 2,
   MEDIUM: 4,
   HEAVY: 6,
 };
+
+const MOUSECODES = {
+  LEFTCLICK: 0,
+};
+
 const propTypes = {
   /**
    * The line width to use when drawing the signature on the canvas.
-   * One of LineWidths.EXTRA_FINE, LineWidths.FINE, LineWidths.MEDIUM, LineWidths.HEAVY.
+   * One of LINEWIDTHS.EXTRA_FINE, LINEWIDTHS.FINE, LINEWIDTHS.MEDIUM, LINEWIDTHS.HEAVY.
```

Comment:
This comment isn't actually accurate, consumers would be using Signature.Opts.Width.EXTRA_FINE etc.